### PR TITLE
Update constants.py

### PR DESCRIPTION
**Pull request by HarshMN2345**
added workfow endpoint which solves the issue #92

### CODE REVIEW

**Comment by HarshMN2345** [comment]
@Cliftonz please review this

File: novu/constants.py [hunk 1]
```diff
@@ -16,5 +17,7 @@
 TENANTS_ENDPOINT = "/v1/tenants"
 TOPICS_ENDPOINT = "/v1/topics"
 BLUEPRINTS_ENDPOINT = "/v1/blueprints"
+WORKFLOW_ENDPOINT = "/v1/workflow"
```
Comment (unicodeveloper) [inline novu/constants.py:20]: This should be `/v1/workflows`

**Comment by Cliftonz** [comment]
@HarshMN2345 Please make sure the CI/CD passes

**Comment by HarshMN2345** [comment]
@Cliftonz all checks have passed

**Review by Cliftonz — APPROVED** [review]

**Pull request merged**